scripts/ci/build-installers.sh

```diff
@@ -46,10 +46,11 @@ fi
 
 if [ "$ARGV_OPERATING_SYSTEM" == "linux" ]; then
   ./scripts/build/docker/run-command.sh \
-    -r "$TARGET_ARCH" \
+    -r "$ARGV_ARCHITECTURE" \
```

Comment:
Haha, I'd also spotted this discrepancy and had this change in a WIP branch that I haven't pushed yet ;-)